src/Components/Alert/Components/Modal.tsx

```diff
@@ -42,6 +42,7 @@ export const Modal: React.FC<AlertModalProps> = ({
         width="100%"
         height="100%"
         position="relative"
+        overflowY="auto"
```

Comment:
@tam-kis Not quite sure what the reason behind [removing this line](https://github.com/artsy/force/pull/13156/files#diff-e2d6453b65cffc01bdde27c288e20b8bfaa3dc5395e48579c9520c066f1eef04L46) was and hope bringing it back doesn't break anything 🤔  @tam-kis

Reply:
This was affecting the sticky header, but now it's working fine. We can add this back 👍